KubeArmor/core/containerdHandler.go

```diff
@@ -132,8 +132,11 @@ func (ch *ContainerdHandler) GetContainerInfo(ctx context.Context, containerID s
 	// == container base == //
 
 	container.ContainerID = res.Container.ID
-	container.ContainerName = res.Container.ID[:12]
-
+	if len(res.Container.ID) < 12 {
+		container.ContainerName = res.Container.ID
+	} else {
+		container.ContainerName = res.Container.ID[:12]
+	}
```

Comment:
Let's not do this.
Let's keep it
```suggestion
	container.ContainerName = res.Container.ID
```

This is because we are only trimming to so that we have a saner logging mechanism. I will comment in the next review what we can do to avoid that.

Reply:
ok, I think is nice, @daemon1024 I have change code to your suggest.